engine/sql/072_workflow_node_run_vcs.sql

```diff
@@ -0,0 +1,8 @@
+-- +migrate Up
```

Comment:
Why not a data column JSONB and use specific json request ?
A nodeRun must not have specific column about repository (that's  why we create workflow :))

Reply:
Discussed with @fsamin about jsonb infos. The fact is that query some data in jsonb could be painful and slow. And considering informations about vcs, if no application is linked we don’t use these data and it stays empty. The conclusion of this solution is that relying on tag to fetch commit and other informations is a bad thing. Tag should stay to inform users and we have not to use these for our process in our code

Reply:
not sure about paintful and slow if we consider a simple map object.
I agree about using tags.